### DISCUSSION TITLE
19 maggio 2021

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#1] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 19 maggio 2021 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[19-05-2021 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988456/19-05-2021.-.COMPLETO.pdf)

**Comment by FeddyLix17** [comment]
[Walkthrough Esame 19 Maggio 2021](https://github.com/FeddyLix17/Computer-Science/tree/main/Primo%20Anno/Secondo%20Semestre/Metodologie%20di%20Programmazione/Esami%20precedenti/2021/Maggio/Esame%2019%20Maggio%202021)
